src/Core/Abstractions/IFido2GetAssertionUserInterface.cs

```diff
@@ -0,0 +1,18 @@
+namespace Bit.Core.Abstractions 
+{
+    public struct IFido2GetAssertionUserInterfaceCredential
+    {
+        public string CipherId { get; set; }
+        public bool RequireUserVerification { get; set; }
+    }
```

Comment:
🎨 Remove `I` prefix given that it's not an interface
```suggestion
    public struct Fido2GetAssertionUserInterfaceCredential
    {
        public string CipherId { get; set; }
        public bool RequireUserVerification { get; set; }
    }
```